### ISSUE TITLE
Add Adobe Latin Character Sets in Encoding menu

### ISSUE THREAD

**Issue by pathumego** [fontforge/fontforge#2182]
Add [Adobe Latin Character sets](http://adobe-type-tools.github.io/adobe-latin-charsets/adobe-latin-1.html) which i a industry standard.

**Comment by davelab6** [comment]
We should support enc files, like https://github.com/tphinney/font-tools

**Comment by pathumego** [comment]
Yep it is a pain.. Encoding Make from font is there. But I don't see any
files saved. It is temporarily saved for the session.

Support for enc would be great.
